subscription.go

```diff
@@ -48,11 +48,16 @@ type SubscriptionInput struct {
 	Name               string      `json:"name"`
 }
 
+type SubscriptionTerminateInput struct {
+	Status string `json:"status,omitempty"`
```

Comment:
I would suggest to add the `ExternalSubscriptionID` here as well to replace the argument in the Terminate method call.